wsagent/che-core-api-debug/src/main/java/org/eclipse/che/api/debugger/server/Debugger.java

```diff
@@ -106,31 +108,80 @@ default void suspend() throws DebuggerException {
   /**
    * Gets the current value of the given variable.
    *
+   * @deprecated
+   * @see #getValue(VariablePath, long, int)
```

Comment:
I think it is better to write 'Use ...' instead of `@see` annotation, especially if it is already done in the last method: https://github.com/eclipse/che/pull/5320/files#diff-892f91fe9e6ba4ea683e504e3a9f4c1cR220